less/index.less

```diff
@@ -81,10 +88,22 @@ header {
         justify-content: space-between;
         display: flex;
 
+        @media( max-width: 800px ) {
+        margin-top: 40px; 
```

Comment:
This media query for example is one area you'd want to use more indentation, from margin-top through display-flex.